src/stores/room-list/algorithms/Algorithm.ts

```diff
@@ -382,7 +382,7 @@ export class Algorithm extends EventEmitter {
 
         if (!this._stickyRoom) {
             // If there's no sticky room, just do nothing useful.
-            if (!!this._cachedStickyRooms) {
+            if (this._cachedStickyRooms) {
```

Comment:
Please can we keep the boolean coercion?

Reply:
Sure!